spec/features/material_search_spec.rb

```diff
@@ -28,7 +28,7 @@
     click_on "Search"
 
     expect(page).not_to have_content("Cheese")
-    expect(page).not_to have_content("Big Pizza")
+    expect(page).to have_content("Big Pizza")
```

Comment:
I would change this and the above check to something like:
```
within(page.find('results selector')) do
  expect(page).not_to have_content("Cheese")
  expect(page).not_to have_content("Big Pizza")
end
```

The purpose of this being here was to ensure that we don't have any results being displayed so it would be good to stick with that rather then mixing in the new dropdown into this as well.